tools/deb-tools/bazel-deb-tools

```diff
@@ -0,0 +1 @@
+/private/var/tmp/_bazel_prifel/1b4f270d25cb3a9340b9e5aad3a4249e/execroot/__main__
```

Comment:
This (symlink?) probably shouldn't be committed.. though it looks like it isn't causing problems...

Reply:
ah thanks, i'm not sure which command I ran that generated that symlink but it should probably get gitignored if its still created after deb-tools is fixed.